charts/zeebe-operate-helm/templates/deployment.yaml

```diff
@@ -20,6 +20,13 @@ spec:
       containers:
       - name: {{ .Chart.Name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        {{- if .Values.extraEnv}}
+        env:
+        {{ .Values.extraEnv | toYaml | nindent 10 }}
```

Comment:
I would suggest we call it just `env` as we do it in the other charts https://github.com/camunda-community-hub/camunda-cloud-helm/blob/main/charts/zeebe-cluster-helm/templates/statefulset.yaml#L87

Reply:
Sure, make sense